code/game/machinery/washing_machine.dm

```diff
@@ -184,11 +184,55 @@ GLOBAL_LIST_INIT(dye_registry, list(
 	var/bloody_mess = FALSE
 	var/obj/item/color_source
 	var/max_wash_capacity = 5
+	var/datum/looping_sound/wash/wash_loop
+	var/door_open_sound = 'sound/vehicles/clown_car/door_open.ogg' //sounds similar enough
+	var/door_close_sound = 'sound/vehicles/clown_car/door_close.ogg'
+
+/obj/machinery/washing_machine/Initialize(mapload)
+	. = ..()
+	register_context()
+	wash_loop = new(src, FALSE)
+	wash_loop.extra_range = -12
+	wash_loop.start_volume = 30
+	wash_loop.end_volume = 30
+
+/obj/machinery/washing_machine/Destroy()
+	QDEL_NULL(wash_loop)
+	return ..()
+
+/obj/machinery/washing_machine/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+	. = ..()
+	if(busy)
+		return NONE
+	if(held_item)
+		return NONE
```

Comment:
potentially include screentips for held item interactions, i.e. opening the panel (screwdriver), anchoring (wrench), putting items in (all the other ones)